### PR TITLE
Put the boots to him, medium style

### PR DESCRIPTION
**Pull request by Kierany9**
Ports stomping.
Magboot stomps have had their windup and cooldown increased by 5 deciseconds.
Due to the lack of our stamina overhaul on TG, stomping simply increases the kick damage by 50% instead instead of turning stamina into brute.

40mm grenades for the c90 have been buffed from (-1, 0, 2) to (-1, 2, 3), as opposed to Hippie's (-1, 3, 2) (which is actually (-1, 3, 3) because the light range can't be lower than the heavy range) and reduced the direct hit damage to 20 instead of 60. This is to give it both its old stopping power and semi-reliable wall destruction. Getting caught within a 3x3 area of an explosion(that didn't hit a wall) is still going to be a death sentence but being clipped by the edge of the blast will no longer knock you out.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -955,23 +955,41 @@
...
 				var/obj/item/bodypart/affecting = H.get_bodypart(ran_zone(M.zone_selected))
+				var/armor_block = H.run_armor_check(affecting, "melee")
+
+				if(H.lying)
+					atk_verb = "kick"
+					if(M.shoes)
+						var/obj/item/clothing/shoes/S = M.shoes
+						if(S.stomp > 0)
+							atk_verb = "stomp"
+							damage *= 1.5
```
Comment (Carbonhell) [inline code/modules/mob/living/carbon/human/species.dm]: won't this also double the damage when stomp is == 2

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -955,23 +955,41 @@
...
+							add_logs(M, H, "stomped")
+							return 1
 
 				if(!damage || !affecting)
 					playsound(H.loc, M.dna.species.miss_sound, 25, 1, -1)
 					H.visible_message("<span class='warning'>[M] has attempted to [atk_verb] [H]!</span>")
 					return 0
 
-
-				var/armor_block = H.run_armor_check(affecting, "melee")
```
Comment (Carbonhell) [inline code/modules/mob/living/carbon/human/species.dm]: ???

File: code/modules/mob/living/carbon/human/species.dm (hunk 1 above)
Comment (Kierany9) [inline code/modules/mob/living/carbon/human/species.dm]: No, stomp = 2 always applies 45 damage.

**Comment by Carbonhell** [comment]
seems k

**Pull request merged**